Capacitor-Banks.md

```diff
@@ -1,36 +1,42 @@
+# Capacitor Banks
 Capacitor Banks store power.  Capacitor Banks of the same tier can connect into a free-form multiblock.  Each capacitor increases the total storage capacity, and the maximum transfer rate.
 
-The Capacitor Bank GUI has four slots to charge RF items, can set the Max Input and Max Output, can react to a redstone signal to enable/disable energy input and output separately, and allows quick access to the Armor, off-hand, and Baubles slots.  By mousing over the power bar you can see precise statistics relating to current power, maximum power, and current net power gain or loss.
+
+![Capacitor Bank GUI](https://github.com/paul-soporan/enderio-wiki/blob/master/images/GUIs/Capacitor-Bank-GUI.png)
```

Comment:
absolute url. shorten it to "images/GUIs/Capacitor-Bank-GUI.png" so it is relative to the document and doesn't point to your fork forever.

Same for the other images you added.